File: Thanos Snap/estalodothanos.go
package main

import (
    "fmt"
    "math/rand"
    "time"
)

func main() {
    
    rand.Seed(time.Now().UnixNano())
    var destiny int = rand.Intn(2)

    if destiny == 0 {
        fmt.Println("You were spared by Thanos.")
    } else {
        fmt.Println("You were slain by Thanos, for the good of the Universe.")
    }
}

File: Thanos Snap/estalodothanos.cpp
#include <cstdlib> 
#include <ctime> 
#include <iostream>
using namespace std;

int main() 
{ 
    srand((unsigned)time(0)); 
    
    int destiny;
    destiny = (rand()%2)+1; 
    
    if (destiny==1){
    cout << "You were spared by Thanos." << endl;
    }
    
    if (destiny==2){
    cout << "You were slain by Thanos, for the good of the Universe." << endl;
    }
    return 0;
}